distribution/src/scripts/forget-me.bat

```diff
@@ -0,0 +1,40 @@
+@echo off
+rem Copyright 2005-2018 WSO2, Inc. (http://wso2.com)
```

Comment:
I think license header should be updated to latest

Reply:
Fixed in commit [1].

[1] 04a93a4737c3f63ad3510e4f8aad907217f94458